src/services/filteringTodos.ts

```diff
@@ -0,0 +1,29 @@
+import { Todo } from '../types/Todo';
+
+const IS_ACTIVE = 'active';
+const COMPLETED = 'completed';
+
+export const filteringTodo = (
+  todoList: Todo[] | null,
+  query: string,
+  selectFiltering: string = '',
```

Comment:
The default value for `selectFiltering` is an empty string. Consider setting a default value like 'all' to align with the filtering options and ensure consistent behavior.